sync.cmd

```diff
@@ -0,0 +1,25 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+set _args=%*
+if /I [%1]==[/?] goto Usage
+if [%1]==[]  set _args=-p
+
+echo Running executor.cmd %_args%
+call executor.cmd %_args%
+if NOT [%ERRORLEVEL%]==[0] (
```

Comment:
I think I could live one a little error handling (so not exactly 1 line) but I think we can get this entire script down to just 
```
call executor.cmd sync %*
exit /b %ERRORLEVEL%
```